### ISSUE TITLE
When user streams from non white listed IP but is in the ignore user list, kill notification sent

### ISSUE THREAD

**Issue by Nostang3** [blacktwin/JBOPS#33]
I have entered my username under the IGNORE_LST, and purposefully streamed from a IP not listed in the IP_WHITELIST. When playback starts a discord notification is sent that playback started and then it will post an IP Whitelist Violation notification yet the stream isn't killed. Which it shouldn't because my username is ignored. 

Here is the logs:
https://pastebin.com/W2a8heqT

**Comment by blacktwin** [comment]
The issue is that the notification is being sent out when it shouldn't be. Odd. I just tested it on myself and it works as intended. Make the following edits to see what is happening:

[Lines 79-87]( https://github.com/blacktwin/JBOPS/blob/master/killstream/ip_whitelist.py#L79-L87)
```python
if ip_address not in IP_WHITELIST:
    print('IP: {ip} not in whitelist.'.format(ip=ip_address))  # <---- Add print
    for session in plex.sessions():
        username = session.usernames[0]
        print('Session Key: {sess}'.format(sess=session.sessionKey)) # <---- Add print
        title = (session.grandparentTitle + ' - ' if session.type == 'episode' else '') + session.title
        if session.sessionKey == int(sessionKey) and username not in IGNORE_LST:
            sys.stdout.write("Killing {user}'s stream of {title}. IP: {ip} not in whitelist".format(
                user=username, title=title, ip=ip_address))
            session.stop(reason=REASON.format(ip_address))
            send_notification(SUBJECT_TEXT, BODY_TEXT.format(user=username, ip=ip_address, title=title))
        else:    # <---- Add else
            print('User: {user} is ignored.'.format(user=username)) # <---- Add print
else:     # <---- Add else
    print('IP: {ip} is in whitelist.'.format(ip=ip_address)) # <---- Add print
```

Show logs but only from time of testing.

**Comment by Nostang3** [comment]
Sorry, for the delay. Long weekend. 

Anyway, what i discovered is that when I use your new lines of code, it works fine. And when I use mine it would produce a IP violation. I got to comparing the code and the only thing that was different is that the last line 
`send_notification(SUBJECT_TEXT, BODY_TEXT.format(user=username, ip=ip_address, title=title))`
was not indented properly on mine. Even when I copy and pasted from github, it puts that line up against the left margin. When I indent it over to match the `session.stop(reason=REASON.format(ip_address))` line it works as expected but creates a unexpected indent error. I fixed that by copying your code over from above and removed the extras then I don't get the indent error. Now everything seems to work fine. 

Summary: Copy and pasting from github messed up indent formatting? If you're satisfied I can close this.
